core/internal/cltest/factories.go

```diff
@@ -509,6 +509,24 @@ func NewEthTx(t *testing.T, store *strpkg.Store, fromAddress common.Address) mod
 	}
 }
 
+func MustInsertEthTxWithBroadcastAttempt(t *testing.T, store *strpkg.Store, nonce int64, fromAddress common.Address, opts ...interface{}) models.EthTx {
```

Comment:
Surprised we didn't already have one of these... what makes it different?

Reply:
oh the name was actually kind of wrong - changed now to MustInsertUnconfirmedEthTx.
The existing one was MustInsertUnconfirmedEthTxWithBroadcastAttempt but it was confusing in a few tests, because they were subsequently inserting more attempts. It's more clear to insert just the TX and attempts separately.